src/components/Community/Contribute/index.tsx

```diff
@@ -113,6 +113,23 @@ const Contribute: React.FC<{ theme: ICommunitySectionTheme }> = ({ theme }) => {
               blog posts, or hosting meetups.
             </CommunityBlock>
           </div>
+          <div className={sharedStyles.item}>
```

Comment:
Hmmm, while we definitely want to include a link that takes you to the contribution docs, I think adding another `CommunityBlock` makes things look too cramped: 

<img width="1182" alt="image" src="https://user-images.githubusercontent.com/43496356/174289380-817c2571-75c7-4cce-b2e8-e74677ffd4a4.png">

Reply:
Instead of adding a new block, could we add a link or two into the text under "Contribute": 

<img width="1269" alt="Screen Shot 2022-06-17 at 6 28 25 AM" src="https://user-images.githubusercontent.com/43496356/174290116-10fd93f1-70f3-4b1b-9971-1c2a747f82ee.png">

For example, instead of "DVC is 100% open source. We welcome contributions to our code and docs" we could do "DVC is 100% open source. We welcome contributions to our [code](https://dvc.org/doc/user-guide/contributing/core) and [docs](https://dvc.org/doc/user-guide/contributing/docs)"

Reply:
Yeah, that's a better idea, 
I'm going to update the PR.

Reply:
That's a better idea. It'll be better looking.

I think instead of 
 "DVC is 100% open source. We welcome contributions to our [code](https://dvc.org/doc/user-guide/contributing/core) and [docs](https://dvc.org/doc/user-guide/contributing/docs)" 

we should explicitly notify the reader to read the docs by saying 
  "DVC is 100% open source. We welcome contributions to our code and docs. Take a look at contribution [code](https://dvc.org/doc/user-guide/contributing/core) and  [docs](https://dvc.org/doc/user-guide/contributing/docs)"

How does that sound?

Reply:
Sounds good! I'm no writer but I think "Take a look at contribution [code](https://dvc.org/doc/user-guide/contributing/core) and [docs](https://dvc.org/doc/user-guide/contributing/docs)" should be "Take a look at contributing to our [code](https://dvc.org/doc/user-guide/contributing/core) and [docs](https://dvc.org/doc/user-guide/contributing/docs)".

Reply:
I'm not a native English speaker so I guess they both kinda sound the same 😅.
I'll go with your suggestion though.

Reply:
I feel like the huge "Contribute" is a pretty clear CTA. Maybe give it a `!` at the end. Then we can keep just "We welcome contributions to our code and docs." which is short and simple.

Reply:
p.s. we could also have a single link to https://dvc.org/doc/contributing (which redirects to core) since the core contrib page starts by mentioning the other ways to contribute. No strong opinion on this. Thanks